src/auth/auth-api-request.ts

```diff
@@ -481,6 +495,21 @@ export const FIREBASE_AUTH_GET_ACCOUNT_INFO = new ApiSettings('/accounts:lookup'
     }
   });
 
+/**
+ * Instantiates the getAccountInfo endpoint settings for use when fetching info
+ * for multiple accounts.
+ */
+export const FIREBASE_AUTH_GET_ACCOUNTS_INFO = new ApiSettings('/accounts:lookup', 'POST')
```

Comment:
Why do you need to duplicate `FIREBASE_AUTH_GET_ACCOUNT_INFO` above? It is the same underlying API with similar request /response format.